CI/requirements.txt

```diff
@@ -1,4 +1,4 @@
 # minimal set of packages for github actions to run our test suites
 codecov
 tox==3.21.2
-virtualenv<20
+virtualenv>=20
```

Comment:
I would recommend a sufficiently newer version of virtualenv 20.x -- the early versions were quite buggy.  the minimum bound in `pre-commit` is a reasonable one: https://github.com/pre-commit/pre-commit/blob/24d9dc7469f9b752b5c772d5cc829d95b43e0e8c/setup.cfg#L31